source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -563,14 +538,236 @@ the Kubernetes cluster.
+
+- :mc:`~kubectl minio tenant create --volumes`
+
+  The total number of storage volumes (Persistent Volume Claims). 
+  The Operator generates an equal number of PVC *plus two* for supporting Tenant services (Metrics and Log Search).
+       
+  The total number of persistent volume claims (``PVC``) per server is determined by dividing the number of volumes by the number of servers.
+  The storage available for each ``PVC`` is determined by dividing the capacity by the number of volumes. 
+
+  The generated claims have pod selectors so that claims are only made for volumes attached to node running the pod.
+
+  If the number of volumes exceeds the numnber of persistent volumes available on the cluster, ``MinIO`` hangs until the number of persistent volumes are available.
+  
+- :mc:`~kubectl minio tenant create --namespace`
+
+  Each MinIO tenant requires its own ``namespace``.
+
+  When not specified, the MinIO Operator defaults to use ``minio``.
+  To specify a different namespace, use the :mc:`~kubectl minio tenant create --namespace` flag.
+
+  The namespace must already exist in the Kubernetes cluster.
+  Run ``kubectl create ns <new_namespace>`` to add one.
+
+.. _create-tenant-cli-determine-additional-options:
+
+2) Determine Additional Optional Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-STUB
\ No newline at end of file
+You can further customize your tenant while executing the command by including any or all of the *optional* flags.
+Determine whether to use each of the following and the value to set:
+
+- :mc:`~kubectl minio tenant create --image`
+           
+  Customize the ``minio`` image to use.
+  
+  By default, the Operator uses the release image available at the time of the Operator's release.
+  To specify a different MinIO version for the tenant, such as the latest available, use the :mc:`~kubectl minio tenant create --image` flag.
+
+  See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.
+
+- :mc:`~kubectl minio tenant create --image-pull-secret`
+           
+  If required, specify the secret to use when pulling the ``minio`` image.
+
+  Use :mc:`~kubectl minio tenant create --image-pull-secret` to specify the secret.
+
+- :mc:`~kubectl minio tenant create --kes-config`
+           
+  Configure a :minio-git:`Key Encrption Service (KES) <kes>`
+
+  Use the :mc:`~kubectl minio tenant create --kes-config` flag to specify the name of the secret to use for KES Key Management Service (KMS) setup.
+
+  Enabling Server Side Encryption (SSE) also deploys a MinIO :minio-git:`KES <kes>` service in the Tenant to faciliate SSE operations.
+  
+  For more, see the `Github documentation <https://github.com/minio/kes/wiki>`__.
+
+- :mc:`~kubectl minio tenant create --storage-class`
```

Comment:
🤔 so this is strictly not *optional*, but I think failing to specify this up front is likely to result in failure.  The user has to make sure the specified storage class exists and has sufficient PVs to bind. I'd move this up.

Reply:
Moved to the required section.